### PR TITLE
Publishes docker image to GitHub package registry.

### PR DESCRIPTION
**Pull request by shinjonathan**

### CODE REVIEW

**Comment by shinjonathan** [comment]
This action publishes any commits to `main` as `image:latest`

You will need to log into the Github docker registry by creating a Github Personal Access Token. I'll document those steps in the front end repository readme.

File: .github/workflows/docker-publish.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    # Publish `v1.2.3` tags as releases.
+    tags:
+      - v*
+
+  # Run tests for any PRs.
+  pull_request:
+
+env:
+  IMAGE_NAME: image
```
Comment (shinjonathan) [inline .github/workflows/docker-publish.yml:17]: Anyone have any thoughts on what we want to call the image name?

docker.pkg.github.com/civictechindex/cti-website-backend/`image`:latest

Comment (shinjonathan) [reply .github/workflows/docker-publish.yml]: Set this image name to service.

**Comment by shinjonathan** [comment]
@Abhishek-AC @cnk 

would you please review this PR.
#11 #14 this will allow the front end project to pull this image to run locally.

**Review by Abhishek-AC — APPROVED** [review]

**Comment by Abhishek-AC** [comment]
> You will need to log into the Github docker registry by creating a Github Personal Access Token. I'll document those steps in the front end repository readme.

@shinjonathan requesting to document these steps.

**Pull request merged**